### PR TITLE
fix: Cherry pick action

### PR DESCRIPTION
**Pull request by pedronfigueiredo**
## **Description**

Fixes the cherry pick action by moving the script to the .github folder as it's expected, instead of the .circleci folder.

Tested in [this job](https://github.com/MetaMask/metamask-extension/actions/runs/8847317364/job/24294909853).

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/24256?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by matthewwalsh0 — DISMISSED** [review]

File: .github/workflows/create-cherry-pick-pr.yml [hunk 1]
```diff
@@ -13,19 +13,18 @@ on:
...
-      run: |
-        ./scripts/create-cherry-pick-pr.sh ${{ github.event.inputs.branch_name }} ${{ github.event.inputs.commit_hash }} ${{ github.event.inputs.PR_number }}
\ No newline at end of file
+      - name: Create Cherry Pick PR
+        id: create-cherry-pick-pr
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./scripts/create-cherry-pick-pr.sh ${{ github.event.inputs.branch_name }} ${{ github.event.inputs.commit_hash }} ${{ github.event.inputs.PR_number }}
```
Comment (matthewwalsh0) [inline .github/workflows/create-cherry-pick-pr.yml:30]: Is this path relative to this `workflows` directory or the `.github` directory?

If the former, would it need to be `../scripts`?

Comment (pedronfigueiredo) [reply .github/workflows/create-cherry-pick-pr.yml:30]: Good point, looking at the mobile repo implementation (which inspired this one and is currently working), this path is relative to the repo root folder. I still think keeping it in the `.github` folder makes sense, so I'm changing the path accordingly.

https://github.com/MetaMask/metamask-mobile/blob/main/.github/workflows/create-cherry-pick-pr.yml#L38-L39
https://github.com/MetaMask/metamask-mobile/blob/main/scripts/create-cherry-pick-pr.sh

**Review by matthewwalsh0 — DISMISSED** [review]

**Review by OGPoyraz — DISMISSED** [review]

**Review by DDDDDanica — DISMISSED** [review]

File: .github/workflows/create-cherry-pick-pr.yml [hunk 2]
```diff
@@ -13,19 +13,28 @@ on:
...
 jobs:
   cherry-pick:
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: write
+      pull-requests: write
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
```
Comment (pedronfigueiredo) [inline .github/workflows/create-cherry-pick-pr.yml:23]: can we revert this change

**Review by vinistevam — DISMISSED** [review]

**Review by brad-decker — DISMISSED** [review]

File: .github/workflows/create-cherry-pick-pr.yml [hunk 3]
```diff
@@ -13,19 +13,27 @@ on:
...
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get Node.js version
+        id: nvm
+        run: echo "NODE_VERSION=$(cat .nvmrc)" >> "$GITHUB_OUTPUT"
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
```
Comment (legobeat) [inline .github/workflows/create-cherry-pick-pr.yml:32]:
```suggestion
      - uses: actions/setup-node@v4
        with:
          node-version-file: .nvmrc
```

https://github.com/actions/setup-node?tab=readme-ov-file#usage

**Review by jpuri — DISMISSED** [review]

**Review by HowardBraham — APPROVED** [review]
